### PR TITLE
Add a service account and a role to configure the AWS access

### PR DESCRIPTION
**Pull request by meln1k**
Added a service account and a role to configure what is allowed to be collected by resoto

### CODE REVIEW

**Comment by TheCatLady** [comment]
Nice! I think the fact that a service account will be created should also be documented in the CDK deployment guide, so that there are no surprises to the user. What do you think?

**Review by aquamatthias — APPROVED** [review]
Nice - thanks!

File: lib/resoto-helm-chart-stack.ts [hunk 1]
```diff
@@ -11,6 +11,38 @@ export class ResotoHelmChartAddOn implements ClusterAddOn {
...
+    const saNamespace = 'default';
+    const saAccountName = 'resoto-helm-chart-sa';
+
+    const openIdProviderCondition = new CfnJson(stack, 'openIdProviderCondition', {
+      value: {
+        [`${openIdConnectProviderIssuer}:sub`]: `system:serviceaccount:${saNamespace}:${saAccountName}`
+      }
+    });
+
+    const resotoServiceAccountIamRole = new iam.Role(stack, 'ResotoServiceAccountIamRole', {
```
Comment (aquamatthias) [inline lib/resoto-helm-chart-stack.ts:26]: Can you also define the name?
The IAM role in the current stack is named: `resoto-eks-stack-test-ResotoServiceAccountIamRole1-PUJJLU1BJU5T`

Comment (meln1k) [reply lib/resoto-helm-chart-stack.ts:26]: done

**Pull request merged**